docs/index.rst

```diff
@@ -2,26 +2,34 @@
 
 .. _plasmapy-documentation:
 
+.. image:: _static/graphic-circular.png
+   :alt: PlasmaPy logo
+   :align: right
+   :scale: 40%
+
 ######################
 PlasmaPy Documentation
 ######################
 
+`PlasmaPy <http://www.plasmapy.org/>`_ is an open source
+community-developed core `Python <https://www.python.org/>`_ 3.7+
+package for plasma physics currently under development.
+
+Example highlights
+------------------
+
 .. nbgallery::
    :hidden:
 
    notebooks/diagnostics/proton_radiography_particle_tracing
    notebooks/dispersion/two_fluid_dispersion
    notebooks/diagnostics/thomson
-
-.. image:: _static/graphic-circular.png
-   :alt: PlasmaPy logo
-   :align: right
-   :scale: 80%
-
-
-`PlasmaPy <http://www.plasmapy.org/>`_ is an open source
-community-developed core `Python <https://www.python.org/>`_ 3.7+
-package for plasma physics currently under development.
+   notebooks/analysis/swept_langmuir/find_floating_potential
+   notebooks/diagnostics/proton_radiography_particle_tracing_wire_mesh
+   notebooks/formulary/thermal_bremsstrahlung
+   notebooks/plasma/grids_nonuniform
+   notebooks/ExB_drift
+   notebooks/simulation/particle_stepper
```

Comment:
```suggestion
   notebooks/analysis/swept_langmuir/find_floating_potential
   notebooks/ExB_drift
   notebooks/simulation/particle_stepper
```
In office hours today, @Tiger-Du, @svincena, and I talked about which notebooks to drop if we went down to six, and we're thinking these three...though it was a tough decision!  

The reason was that if viewed on a mobile device, the notebooks often show up all in a single column so you have to scroll down past all of them.